pandas/tests/io/test_pytables.py

```diff
@@ -5167,3 +5168,70 @@ def test_dst_transitions(self):
                 store.append('df', df)
                 result = store.select('df')
                 assert_frame_equal(result, df)
+
```

Comment:
can you move these to a separate (new) file: test_pytables_compat.py (even better to move to

pandas/tests/io/pytables/test_compat.py (you will need to add an __init__.py in the subdir)

we are starting to split the test_pytables up a bit

Reply:
Doing so.

> (you will need to add an init.py in the subdir)

Why?

It does not need to be a Python package for pytest to discover tests in the directory:
```
$ pytest --collect-only pandas/tests
[...]
<Module pandas/tests/io/pytables/test_compat.py>
  <Class TestReadPyTablesHDF5>
      <Function test_read_complete>
      <Function test_read_with_start>
      <Function test_read_with_stop>
      <Function test_read_with_startstop>
[...]
```
Any other reason?